docs/src/pages/components/slider/slider.md

```diff
@@ -14,6 +14,12 @@ Sliders reflect a range of values along a bar, from which users may select a sin
 
 {{"component": "modules/components/ComponentLinkHeader.js"}}
 
+## Sizes
+
+For smaller slider, use the prop `size="small"`.
+
+{{"demo": "pages/components/slider/SliderSizes.js"}}
```

Comment:
What do you think about rendering this after `## Continuous sliders`. It seems too bold to render first.